### PR TITLE
feat: replace react-helmet with Gatsby Head API

### PR DESCRIPTION
**Pull request by yasell**
This pull request replaces react-helmet with [Gatsby Head API](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-head/)

### CODE REVIEW

File: src/templates/docs/breadcrumb-stub.js [hunk 1]
```diff
@@ -79,4 +72,17 @@ export default function BreadcrumbsStubPage(props) {
...
-}
+};
+
+export default BreadcrumbsStubPage;
+
+export const Head = ({ pageContext: { title } }) => {
+  const pageMetaData = {
+    data: {
+      title,
+      description: ' ',
```
Comment (sashasushko) [inline src/templates/docs/breadcrumb-stub.js:83]: Why did you use the space?

File: src/components/shared/seo/seo.view.js [hunk 2]
```diff
@@ -86,58 +94,49 @@ export const SEO = ({
...
+        />
+      )}
+      {pageTranslations && pageTranslations.es && (
+        <link
+          hrefLang="es"
+          href={`${hrefLangAttributes.es.href}`}
+          rel="alternate"
+        />
+      )}
+      {pageTranslations && pageTranslations.en && (
```
[the remark this answers is not in the document]
Comment (yasell) [reply src/components/shared/seo/seo.view.js]: We want to show the `x-default hreflang` value, that signals that page doesn't target a specific language or locale. Also does e.g. https://reactjs.org/:
Screenshot 2022-11-23 at 15 44 11

File: src/templates/doc-page.js [hunk 3]
```diff
@@ -112,3 +104,27 @@ export default function DocPage(props) {
...
+  pageContext: {
+    remarkNode: { frontmatter },
+    pageVersions,
+  },
+}) => {
+  const pageMetaData = {
+    data: {
+      title: frontmatter.head_title || frontmatter.title,
+      description: frontmatter.excerpt,
+      slug: frontmatter.slug ? frontmatter.slug : location.pathname.slice(1),
```
[the remark this answers is not in the document]
Comment (yasell) [reply src/templates/doc-page.js:121]: location.pathname always starts with `/`, including locale versions

File: src/templates/docs/breadcrumb-stub.js (hunk 1 above)
Comment (yasell) [reply src/templates/docs/breadcrumb-stub.js:83]: This is black box for me:D I left it as it was before

**Review by sashasushko — APPROVED** [review]

**Review by bolotskydev — APPROVED** [review]

**Pull request merged**